README.md

```diff
@@ -318,6 +318,40 @@ $ lerna run build
 
 Run an [npm script](https://docs.npmjs.com/misc/scripts) in each package that contains that script.
 
+`lerna run` respects the `--concurrency` flag (see below).
+
+`lerna run` respects the `--scope` flag (see below).
+
+```sh
+$ lerna run --scope my-component test
+```
+
+### exec
+
+```sh
+$ lerna exec -- [command] // runs the command in all packages
```

Comment:
You are right, I just copied from the run command - both places fixed in de1828c